woodwork/utils.py

```diff
@@ -646,23 +646,32 @@ def _infer_datetime_format(dates, n=100):
         mode_fmt = fmts.mode().loc[0]  # select first most common format
     except KeyError:
         check_for_two_digit_years = [
-            "%m/%d/%y",
             "%y/%m/%d",
-            "%m/%d/%y %H:%M:%S",
```

Comment:
How come these were removed?

Reply:
It's because I add on the timestamps below in a different way that makes sure every format is paired up with a timezone rather than only a few